pkg/pillar/types/global.go

```diff
@@ -220,6 +220,10 @@ const (
 	// ports for image downloads.
 	DownloadMaxPortCost GlobalSettingKey = "network.download.max.cost"
 
+	// BlobDownloadMaxRetries global setting key
+	// how many times EVE will retry to download a blob if its checksum is not verified
+	BlobDownloadMaxRetries GlobalSettingKey = "blob.download.max.retries"
+
```

Comment:
It should be documented in https://github.com/lf-edge/eve/blob/master/docs/CONFIG-PROPERTIES.md.

Reply:
Thank you @OhmSpectator.